CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contributing to osbuild-composer
+
+First of all, thank you for taking the time to contribute to osbuild-composer. In this document you will find information that can help you with your contribution.
+
+### Running from sources
+
+We recommend using the latest stable Fedora for development but latest minor release of RHEL/CentOS 8 should work just fine as well. To run osbuild-composer from sources, follow these steps:
+
+```
+$ git clone https://github.com/osbuild/osbuild-composer.git
+$ cd osbuild-composer
+$ dnf group install 'RPM Development Tools' # Install rpmbuild
+$ dnf builddep golang-github-osbuild-composer.spec # Install build-time dependencies
+$ make rpm
+$ sudo dnf install rpmbuild/RPMS/x86_64/* -y
+$ sudo dnf install cockpit-composer -y
+$ sudo systemctl enable --now osbuild-composer.socket
+$ sudo systemctl enable --now cockpit.socket
```

Comment:
Why `enable`? Shouldn't this be `start`?

Reply:
`--now` makes it start right away, but it is mostly a personal preference, I want it to start immediatelly after rebooting.